packages/sdk/src/fee/FeeEstimator.ts

```diff
@@ -0,0 +1,144 @@
+import { Psbt } from "bitcoinjs-lib"
+
+import { AddressFormats, getNetwork, getScriptType } from ".."
+import { Network } from "../config/types"
+import { MAXIMUM_FEE } from "../constants"
+import { FeeEstimatorOptions } from "./types"
+
+export default class FeeEstimator {
+  feeRate: number
+  network: Network
+  psbt!: Psbt
+  witness?: Buffer[] = []
+  fee = 0
+
+  private virtualSize = 0
+  private weight = 0
+
+  constructor({ feeRate, network, psbt, witness }: FeeEstimatorOptions) {
+    if (feeRate < 0 || !Number.isSafeInteger(feeRate)) {
```

Comment:
Can't we use BigNumber for SDK? So we have proper precision - https://mikemcl.github.io/bignumber.js/

Reply:
Adding BigNumber support is on our list. Planning to integrate it in a few weeks.